ldk/javascript/src/browser/index.ts

```diff
@@ -57,6 +69,17 @@ export interface Browser {
    */
   listenNetworkActivity(callback: (details: NetworkActivityDetails) => void): Promise<Cancellable>;
 
+  /**
+   * Calls the callback on any UIElement event pushed from a browser running the Olive Helps extension.
```

Comment:
"Calls the callback when an event is pushed from the provided UIElement in a browser running the olive Helps extension."